src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -173,9 +173,14 @@ private void ViewLoadedHandler(object sender, RoutedEventArgs e)
         private void ViewModel_RequestZoomToFit(BoundingBox bounds)
         {
             var prevcamDir = watch_view.Camera.LookDirection;
-            watch_view.ZoomExtents(bounds.ToRect3D(.05));
+            var camera = watch_view.Camera as HelixToolkit.Wpf.SharpDX.PerspectiveCamera;
```

Comment:
it's unlikely but an extension could potentially set the camera in ortho mode, will this change the type of camera? Maybe it's just best to do a check with pattern match:

if( watch_view.Camera is HelixToolkit.Wpf.SharpDX.PerspectiveCam perspectiveCam){
}

Reply:
Makes sence